ios/airship_flutter/Sources/airship_flutter/AirshipPlugin.swift

```diff
@@ -424,8 +424,7 @@ public class AirshipPlugin: NSObject, FlutterPlugin {
         case "preferenceCenter#getConfig":
             return try await AirshipProxy.shared.preferenceCenter.getPreferenceCenterConfig(
                 preferenceCenterID: try call.requireStringArg()
-            )
-            
+            ).unWrap()
```

Comment:
probably dont need to unwrap it but its fine

Reply:
it was throwing a runtime exception